src/game_ui.c

```diff
@@ -3581,191 +3581,192 @@ void hud_render_general(Gfx **dList, MatrixS **mtx, Vertex **vtx, s32 updateRate
     *mtx = gHudMtx;
     *vtx = gHudVtx;
-    if (!(gHudLevelHeader->unkBC & 1)) {
-        sp113 = TRUE;
-        for (i = 0; i < objectCount; i++) {
-            someRacer = (Object_Racer *) objectGroup[i]->unk64;
-            if (someRacer != NULL && someRacer->playerIndex != PLAYER_COMPUTER && someRacer->raceFinished == FALSE) {
-                sp113 = FALSE;
-            }
-        }
-        if ((gHUDNumPlayers == THREE_PLAYERS && is_postrace_viewport_active()) || check_if_showing_cutscene_camera() ||
-            sp113) {
-            goto test;
-        } else if (gHudToggleSettings[gHUDNumPlayers] != 1) {
-        test:
-            return;
-        } else {
-            rendermode_reset(&gHudDL);
-            set_ortho_matrix_view(&gHudDL, &gHudMtx);
-            lvlMdl = get_current_level_model();
-            if (lvlMdl == NULL) {
-                return;
-            }
-            someObjSeg = get_active_camera_segment();
-            sprite_anim_off(TRUE);
-            minimap = (Sprite *) lvlMdl->minimapSpriteIndex;
-            switch (gHUDNumPlayers) {
-                case TWO_PLAYERS:
-                    gMinimapScreenX = 135;
-                    gMinimapScreenY = -gMinimapDotOffsetY / 2;
-                    break;
-                case THREE_PLAYERS:
-                    if (get_current_level_race_type() == RACETYPE_CHALLENGE_EGGS ||
-                        get_current_level_race_type() == RACETYPE_CHALLENGE_BATTLE ||
-                        get_current_level_race_type() == RACETYPE_CHALLENGE_BANANAS) {
-                        gMinimapScreenX = (gMinimapDotOffsetX / 2) - 8;
-                        gMinimapScreenY = -gMinimapDotOffsetY / 2;
-                    } else {
-                        gMinimapScreenX = (gMinimapDotOffsetX / 2) + 72;
-                        gMinimapScreenY = -60 - (gMinimapDotOffsetY / 2);
-                    }
-                    break;
-                case FOUR_PLAYERS:
-                    gMinimapScreenX = (gMinimapDotOffsetX / 2) - 8;
-                    gMinimapScreenY = -gMinimapDotOffsetY / 2;
-                    break;
-                default:
-                    gMinimapScreenX = 135;
-                    gMinimapScreenY = -98;
-                    break;
-            }
-            if (osTvType == OS_TV_TYPE_PAL) {
-                gMinimapScreenY *= 1.2;
-            }
-            sprite_opaque(FALSE);
-            hudElem.pos.x = gMinimapScreenX + gHudOffsetX + gHudBounceX;
-            hudElem.pos.y = gMinimapScreenY;
-            if (osTvType == OS_TV_TYPE_PAL) {
-                hudElem.pos.x -= 4.0f;
-            }
-            hudElem.rotation.z_rotation = -someObjSeg->trans.rotation.z_rotation;
-            hudElem.rotation.x_rotation = 0;
-            if (get_filtered_cheats() & CHEAT_MIRRORED_TRACKS) {
-                hudElem.rotation.y_rotation = -0x8000;
-                hudElem.pos.x -= gMinimapDotOffsetX;
+    if ((gHudLevelHeader->unkBC & 1)) {
```

Comment:
no match change for this function, just applying early return(s) for improved readability